notebooks/CubevizExample.ipynb

```diff
@@ -9,16 +9,52 @@
     "This notebook demonstrates the Cubeviz API in the Notebook setting. UI equivalents for these actions, as well as additional documentation about Cubeviz, can be found here: https://jdaviz.readthedocs.io/en/latest/cubeviz/"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Import modules needed for this notebook."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from jdaviz import Cubeviz\n",
+    "import warnings\n",
+    "\n",
+    "from astropy import units as u\n",
+    "from astropy.coordinates import SkyCoord, Angle\n",
+    "from astropy.utils.data import download_file\n",
+    "from photutils import CircularAperture, SkyCircularAperture\n",
+    "from regions import PixCoord, CirclePixelRegion, CircleSkyRegion\n",
```

Comment:
I'd consider all of this stuff (introduced in this PR) to be pretty advanced for the example notebook (which I think of as just the essentials for someone to get started and build their own notebook)... would the concept notebooks be more appropriate?   Or should we slowly start making these more comprehensive with examples of all the functionality?

Reply:
Maybe should ask PO. I copied content over from Imviz example notebook and I think @camipacifici said such examples were very useful for Imviz.

Reply:
If we decide to remove the sky regions from the notebook, then the appropriate imports should also be removed.